<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proyecto1Organizacion</class>
 <widget class="QMainWindow" name="Proyecto1Organizacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto1Organizacion</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="LabelTitulo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mis Datos del Disco</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNombre">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre del Disco:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParticion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Particion del Disco:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEspacio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Espacio del Disco:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeltamanio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamanio del Disco:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>*******</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>*******</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>*******</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>*******</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonCrear">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonAbrir">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonEliminar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNombre_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre del Disco:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParticion_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Particion del Disco:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEspacio_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Espacio del Disco:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeltamanio_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de Bloques:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNombre">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParticion">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTamanio">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxEspacio">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>256</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>512</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1024</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Mi Disco</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDisco">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disco a Abrir:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDisco">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonCargar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetPint">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>231</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonLog">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
